4C Integration.md

```diff
@@ -0,0 +1,262 @@
+# 4C integration doc
+
+###### This document will give a brief overview of the MX media-buying platform, and explain how to interact with its API.
+
+### Platform Overview
+
+#### Domain Concepts
+
+Within the MX platform we think in terms of `Organizations`, `Users`, `Teams`, `Roles`, `Markets`, `Orders`, `Assets`, `Attributes`, and `Matches`.
+
+* An `Organization` represents a single buyer or seller, and is analogous to a company account. There are many `Users` within an `Organization`, and any action a user takes looks from the outside as if it was done by the organization. Users are assigned to `Teams`, and given `Roles` within those teams which constrain which actions they are allowed to take.
+
+* A `Market` is an environment in which transactions happen. There is an `Open Market` and several `Private Markets`. Initially, all transactions will happen in the latter. `Orders` in private markets can only be seen by market members, which are `Organizations`, and defined by the market owner.
+
+* An `Order` is how we represent supply or demand entered into a `Market`. An order represents the intention of an `Organization` to buy/sell some amount of a certain `Asset` in a `Market`, at some price.
+
+* An `Asset` is what we call media that is bought and sold. It is composed of `Attributes` and additional information, like the date and time. An `Attribute` represents some piece of information about the asset. `Attributes` in turn have `Types`, which can be thought of as classifications of information about assets. This is displayed in the UI in the format of `Type: Attribute Value`, ie, `Network: AMC`.
```

Comment:
maybe this paragraph before the one above. Define asset before it is used in order?

Reply:
i was considering that, but settled on this order, as i felt it was important to define `Order` right after `Market`, and that inserting `Asset` in between would be a non-sequitor. I figured that since `Asset` is defined immediately after the first time its used, it would be fine. what do you think?

Reply:
that's a good point, agreed.